spec/cucumber/rails/database_spec.rb

```diff
@@ -1,18 +1,15 @@
 # frozen_string_literal: true
 
+require 'database_cleaner'
 require 'cucumber/rails/database/strategy'
 require 'cucumber/rails/database/deletion_strategy'
 require 'cucumber/rails/database/shared_connection_strategy'
 require 'cucumber/rails/database/truncation_strategy'
 require 'cucumber/rails/database'
 
 describe Cucumber::Rails::Database do
-  before { allow(strategy_type).to receive(:new).and_return(strategy) }
-
-  let(:strategy) { instance_double(strategy_type, before_js: nil, before_non_js: nil) }
-  let(:strategy_type) { Cucumber::Rails::Database::TruncationStrategy }
-
   context 'when using a valid pre-determined strategy' do
+    let(:strategy) { described_class.instance_eval { @strategy } }
```

Comment:
Not sure for the idea for changing the specs around?

Reply:
I changed this so the specs will use a real strategy instance instead of an `instance_double`.

Reply:
I've dug in and think I can spot why it was weird. Just not keen on instance_eval in specs. But I see it's because we don't allow people to access the strategy value from the specification.

In that case it's all fine. You can move up the strategy let from both blocks to avoid duplication, but other than that it's good.